code/modules/projectiles/projectile/energy.dm

```diff
@@ -114,14 +114,16 @@
 	weaken = 5
 	stun = 5
 
-/obj/item/projectile/energy/sonic/on_hit(var/atom/target, var/blocked = 0)
-	var/mob/M = target
-	if(isturf(target))
-		target.ex_act(0)
-	if(ismob(target))
-		explosion(target, -1, 0, 2)
-		M.gib()
-	..()
+	on_impact(var/atom/A)

```

Comment:
Change this.
`/obj/item/projectile/energy/sonic/on_impact(var/atom/A)`

The rest of the file is formatted like this, so please keep it to the convention. Means you'll also need to tab the code back one.